package.json

```diff
@@ -34,6 +34,10 @@
     {
       "name": "Sing Li",
       "email": "sing.li@rocket.chat"
+    },
+    {
+      "name": "Alex Terpyakov",
+      "email": "terpyakov@gmail.com"
```

Comment:
Thanks for your interest in joining the core team, but we only allow people with considerable contributions to become part of the core team. I hope you understand.